core/runtime/impl/runtime_api.hpp

```diff
@@ -22,10 +27,10 @@ namespace kagome::runtime {
   class RuntimeApi {
    public:
     RuntimeApi(std::shared_ptr<runtime::WasmProvider> wasm_provider,
-               std::shared_ptr<extensions::Extension> extension)
+               std::shared_ptr<extensions::ExtensionFactory> extension_factory)
         : wasm_provider_(std::move(wasm_provider)),
-          memory_(extension->memory()),
-          executor_(std::move(extension)) {}
+          extension_factory_(std::move(extension_factory)),
+          executor_{} {}
```

Comment:
Maybe we should init executor

Reply:
It's constructor changed so perhaps we don't.